bikram.md

```diff
@@ -0,0 +1,95 @@
+1. API Authentication:
```

Comment:
Please use [markdown](https://guides.github.com/features/mastering-markdown/) heading (`#`) here for better readability.